lib/_stream_writable.js

```diff
@@ -287,6 +287,13 @@ Writable.prototype.write = function(chunk, encoding, cb) {
     encoding = null;
   }
 
+  if (this.destroyed) {
+    if (typeof cb === 'function') {
+      process.nextTick(cb, new ERR_STREAM_DESTROYED('write'));
+    }
+    return false;
```

Comment:
Shouldn’t `writeAfterEnd()` have precedence over this?

Reply:
Maybe?